src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -1,41 +1,59 @@
 package nextstep.subway.line.domain;
 
-import nextstep.subway.common.BaseEntity;
-import nextstep.subway.line.dto.LineRequest;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
-import javax.persistence.*;
+import nextstep.subway.common.BaseEntity;
+import nextstep.subway.section.domain.Section;
+import nextstep.subway.section.domain.Sections;
 
 @Entity
 public class Line extends BaseEntity {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-    @Column(unique = true)
-    private String name;
-    private String color;
-
-    public Line() {
-    }
-
-    public Line(String name, String color) {
-        this.name = name;
-        this.color = color;
-    }
-
-    public void update(Line line) {
-        this.name = line.getName();
-        this.color = line.getColor();
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getColor() {
-        return color;
-    }
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	@Column(name = "name", unique = true)
+	private String name;
+	@Column(name = "color")
+	private String color;
+
+	@Embedded
+	private Sections sections = new Sections();
+
+	public Line() {
+	}
+
+	public Line(String name, String color) {
+		this.name = name;
+		this.color = color;
+	}
+
+	public void update(Line line) {
+		this.name = line.getName();
+		this.color = line.getColor();
+	}
+
+	public Long getId() {
+		return id;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public String getColor() {
+		return color;
+	}
+
+	public Sections getSections() {
+		return sections;
+	}
+
+	public void addSection(Section section) {
+		section.changeLine(this);
```

Comment:
외부의 `Section` 참조를 받아서 `Line` 내부에서 호출하고 있네요, 혹시 아래와 같이 구현할순 없을까요?

```suggestion
	public void addSection(Section section) {
		sections.addSection(section);
```

Reply:
section이 추가될때 line 이 함께 추가되어야 된다고 생각했는데 생성시에 이미 line이 들어가니 필요없겠네요.. 감사합니다.